ext_mongodb.php

```diff
@@ -1338,6 +1387,16 @@ public function serialize() : string
 		] );
 	}
 
+	public function jsonSerialize() : mixed
+	{
+		return [
+			'timestamp' => [
+				't' => $this->timestamp,
+				'i' => $this->increment,
```

Comment:
These should be integers according to https://docs.mongodb.com/v3.2/reference/mongodb-extended-json/#timestamp.

According to http://stackoverflow.com/a/13502497/162228, the JSON spec itself does not have a limit on integer sizes. That said, I'm not sure how we'll support this on 32-bit PHP platforms.